### PR TITLE
Added Method for Filtering Shapes/Not Shapes

### PR DESCRIPTION
**Pull request by sshh12**
Added a simple method for ensuring that the contours are at least somewhat shape like. The current constants (2, .65) minimize false negatives.

### CODE REVIEW

File: target_finder/preprocessing.py [hunk 1]
```diff
@@ -82,6 +81,33 @@ def find_blobs(image, min_width=20, max_width=100, limit=100, padding=20):
...
+    """Verify that this could be a blob containing a shape"""
+    width = blob.width
+    height = blob.height
+
+    # check provided restrictions
+    if min(width, height) < min_width or max(width, height) > max_width:
+        return False
+
+    # must be closed cnt
+    # if not blob.has_mask:
```
Comment (bbridges) [inline target_finder/preprocessing.py]: What was this check originally for?

Comment (paigenwilliams) [reply target_finder/preprocessing.py]: The whole thing or just the commented out part?

Comment (bbridges) [reply target_finder/preprocessing.py]: Yeah the commented out part

Comment (paigenwilliams) [reply target_finder/preprocessing.py]: Ah yeah the close contour story is a dependency for that, I told Shrivu to just comment it out for now until that gets sorted. It’s just a check to make sure it’s a fully closed contour that we can apply a mask to.

Comment (bbridges) [reply target_finder/preprocessing.py]: @sshh12 Can you leave a TODO comment on there saying to uncomment once the close contour story is done?

**Pull request merged**